scss/_utilities_content-align.scss

```diff
@@ -33,12 +34,20 @@
   }
 }
 
+// Bottom align flex elements
 @mixin vf-u-align--bottom {
   .u-align--bottom {
     margin-top: auto !important;
   }
 }
 
+// Vertically align inline elements
+@mixin vf-u-align--middle {
+  .u-align--middle {
```

Comment:
It may be a bit confusing that we have `u-align--bottom` (that handles aligning with flexbox using margin) and `u-align--middle` that works in completely different context using completely different props (`vertical-align`).

I'd argue that the new `u-align--middle` is actually less confusing than the existing `u-align--bottom` (that doesn't seem to suggest by its name what context it's meant for).

But not sure how best to address this inconsistency.